OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectWorld.cs

```diff
@@ -64,7 +64,8 @@ private bool DisplayIPv6
             dreamObject.SetVariable("log", log);
 
             DreamValue fps = dreamObject.ObjectDefinition.Variables["fps"];
-            if (fps.TryGetValueAsInteger(out var fpsValue)) {
+
+            if (fps.TryGetValueAsInteger(out var fpsValue) && !_netManager.IsClient) {
```

Comment:
It'd be better to check that this is running on the server, instead of just anything but the client.
```suggestion
            if (fps.TryGetValueAsInteger(out var fpsValue) && _netManager.IsServer) {
```

Reply:
oh didnt know that existed